javascriptv3/example_code/s3/package.json

```diff
@@ -13,17 +13,20 @@
     "@aws-crypto/sha256-browser": "^5.2.0",
     "@aws-doc-sdk-examples/lib": "^1.0.1",
     "@aws-sdk/client-s3": "^3.664.0",
+    "@aws-sdk/crc64-nvme-crt": "^3.731.0",
     "@aws-sdk/credential-providers": "^3.664.0",
     "@aws-sdk/lib-storage": "^3.664.0",
     "@aws-sdk/s3-request-presigner": "^3.664.0",
     "@aws-sdk/util-format-url": "^3.664.0",
     "@smithy/hash-node": "^3.0.7",
     "@smithy/protocol-http": "^4.1.4",
     "@smithy/url-parser": "^3.0.7",
+    "dotenv": "^16.4.7",
     "fast-xml-parser": "^4.5.0",
     "libs": "*"
   },
   "devDependencies": {
+    "prettier": "^3.4.2",
```

Comment:
Prettier is included at the top level js package.json, it's not needed here.

Reply:
removed